tests/schemas/valid/test_placeholder_transform.json

```diff
@@ -4,12 +4,9 @@
   "schema_version": "0.0.1",
```

Comment:
How come this entire schema has changed?. It has other placeholders it was testing.